pylint/testutils/functional/test_file.py

```diff
@@ -62,7 +62,7 @@ class FunctionalTestFile:
     def __init__(self, directory: str, filename: str) -> None:
         self._directory = directory
         self.base = filename.replace(".py", "")
-        # TODO: 2.x: Deprecate FunctionalTestFile.options and related code
+        # TODO: 3.0: Deprecate FunctionalTestFile.options and related code
         # We should just parse these options like a normal configuration file.
```

Comment:
What did you mean by that @DanielNoord ? Just removing self.options is not possible there's the line in `` self.options[name] = conv(value)  # type: ignore[literal-required]`` not sure what to do exactly.

Reply:
We shouldn't need to rely on `options` but rather just have the test run `pylint test_file.py --rcfile=test_file.rc` We have similar logic for the doc tests.

Reply:
Ha right, thank you.